src/app/components/regions/pages/new/new.component.ts

```diff
@@ -71,6 +71,7 @@ NewComponent.linkToRoute({
   resolvers: {
     [projectKey]: projectResolvers.show,
   },
+  title: () => "New Region",
```

Comment:
_region_ is not a UI term:

```suggestion
  title: () => "New Site",
```